recipes/pybda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pybda" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 541908e3874550de3fdceb118279e7530180e1fe9caf7319fe0e29408227f788
+
+build:
+  number: 0
+  skip: true # [not py35 and not py37]
+
+requirements:
+  host:    
+    - python
+    - pip
+    - numpy
+    - setuptools
+  run:
+    - python ==3.6
+    - click >=6.7
+    - joypy >=0.1.10
+    - matplotlib >=2.2.3
+    - numpy >=1.15.0
+    - pandas >=0.23.3
+    - pyspark ==2.4.0    
+    - scipy >=1.0.0
+    - seaborn >=0.9.0
+    - snakemake >=5.2.2
+    - sparkhpc >=0.3.post4
+
+test:  
+  imports:
+    - pybda
+  commands:
+    - pybda --help
+
+about:
+  home: https://github.com/cbg-ethz/pybda
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  summary: Big biological data analytics powered by Apache Spark
+  doc_url: https://pybda.readthedocs.io/en/latest/
+  dev_url: https://github.com/cbg-ethz/pybda
+
+extra:
+  recipe-maintainers:
+    - dirmeier
```

Comment:
You can remove the `extra:` section, we don't end up using it in bioconda (not that it's hurting anything).